meinberlin/assets/scss/components/_switch.scss

```diff
@@ -12,36 +12,38 @@
     }
 
     label:before {
-        background: rgb(0, 0, 0);
-        box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.5);
-        border-radius: 8px;
+        background: $gray;
+        // box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.5);
+        border-radius: 16px;
         content: '';
-        height: 16px;
-        margin-top: -8px;
+        height: 30px;
+        margin-top: -15px;
         position: absolute;
-        opacity: 0.3;
+        // opacity: 0.3;
         transition: all 0.4s ease-in-out;
-        width: 40px;
+        width: 47px;
+        border: 1px solid $gray;
     }
 
     label:after {
-        background: rgb(255, 255, 255);
+        background: $body-bg;
         border-radius: 16px;
-        box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+        // box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
```

Comment:
have commented this stuff as final design for switch not in zeplin, only wireframe, so might need some of this